components/molecule/collapsible/src/_settings.scss

```diff
@@ -0,0 +1,14 @@
+$bgc-collapsible-container: $c-white !default;
+$bgc-collapsible-container-transparent: rgba($bgc-collapsible-container, 0);
+$bg-collapsible-container-gradient: linear-gradient(
+  to top,
+  $bgc-collapsible-container 0%,
+  $bgc-collapsible-container-transparent 100%
+);
+$c-collapsible-btn: $c-primary !default;
+$c-collapsible-icon: $c-collapsible-btn !default;
+$h-collapsible-container-gradient: 48px !default;
+$mh-collapsible-btn: 24px !default;
+$mt-collapsible-btn: 16px !default;
```

Comment:
You should use sui-theme variables as much as possible.

Reply:
Padding and margin with sui-theme variables ✔️
Should I add min-height and height in sui-theme? Or are they ok set here?